### PR TITLE
[DE] Breach Scanner and Mimicry

### PR DESCRIPTION
**Pull request by Red-Z**
# **SCP:SL Translation PR Template**

## **Language Name**
German

## **State Type of change (New language, changed existing Etc.)**
Changed existing files for 939 and 079 HUD according to Hubert's changes in PR #1288 (Adding breach scanner and new Mimicry menu). Also badges update reflecting PR #1294 and Loading Hints #1303

# **Checklist:**
- [x] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  *( Exception to this is when edits has __only__ been made in-browser. If this is the case, also* ***check this box*** *)*
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure my addition follows guidelines described in the [CONTRIBUTING.md](https://github.com/northwood-studios/SCPSL-Translations/blob/master/.github/CONTRIBUTING.md) file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

**Comment by Dr-Ham** [comment]
can't accept it. Only Joseph can

File: Translations/de/LoadingHints.txt [hunk 1]
```diff
@@ -1,63 +1,84 @@
...
-Die Entrance Zone hat zwei Tore die Zugang zur Oberfläche geben. Du benötigst eine Zugangskarte, um sie zu öffnen.
-Fahrstühle sind ein schneller und einfacher Weg SCPs zu entkommen.
+Das Sicherheitspersonal ist alleine schwach, aber gemeinsam stark. Haltet zusammen!
+Deine Ausdauer ist eine kostbare Resource, reserviere ein wenig für Notfälle!
+Die Entrance Zone hat zwei Tore, die Zugang zur Oberfläche ermöglichen. Du benötigst eine Zugangskarte, um sie zu öffnen.
+Fahrstühle sind ein schneller und einfacher Weg, SCPs zu entkommen.
 Entkommenes Klasse-D Personal tritt der Chaos Insurgency bei, während Wissenschaftler den MTF beitritt.
 Adrenalin kann genutzt werden, um deine Ausdauer sofort zu erneuern, es gibt dir außerdem noch einen Vorteil im Kampf.
-Radios haben 4 Reichweiten. Umso weiter die Reichweiter, desto schneller wird die Batterie aufgebraucht.
+Radios haben vier Reichweite-Einstellungen. Umso höher die Reichweiter, desto schneller wird die Batterie aufgebraucht.
```
Comment (Dat-Joseph) [inline Translations/de/LoadingHints.txt:10]: Reichweite has an r at the end.

File: Translations/de/LoadingHints.txt [hunk 2]
```diff
@@ -1,63 +1,84 @@
...
+Tesla Gates sind elektrische Gefahrenstellen, die überall in der Heavy Containment Zone zu finden sind. Passiere sie mit Vorsicht!
 Mobile Task Force Einheiten und Chaos Insurgents erreichen die Surface Zone auf gegenseitigen Seiten.
-Der Alpha Sprengkopf tötet alles und jeden in der Einrichtung bei der Detonation. Er kann auf der Surface Zone aktiviert werden.
-Die Alpha Sprengkopf Detonationssequenz kann aus seinem Raum in der Heavy Containment Zone abgebrochen werden.
-Der Intercomraum in der Entrance Zone kann dazu genutzt werden Sprachnachrichten an alle Spieler zu senden.
-Du kannst Gegenstände werfen ("T" ist Standard). Damit kannst du Zugangskartentüren aus Distanz öffnen oder Glas zerschlagen.
-Die Light Containment Zone wird automatisch nach 15 Minuten verschlossen; jeder in ihr wird dann getötet. Entkomme so lange du kannst!
-Maximiere deinen Vorteil indem du Waffenaufsätze der Situation bedingt justierst.
+Der Alpha-Sprengkopf tötet alles und jeden in der Einrichtung bei der Detonation. Er kann auf der Surface Zone aktiviert werden.
+Die Detonationssequenz des Alpha-Sprengkopfes kann in seinem Raum in Heavy Containment abgebrochen werden.
```
Comment (Dat-Joseph) [inline Translations/de/LoadingHints.txt:14]: We probably need to adjust "seinen", not sure if we assign the male genus to the alpha warhead. Would "dessen" work better?

File: Translations/de/LoadingHints.txt [hunk 3]
```diff
@@ -1,63 +1,84 @@
...
+Während SCP-268 dich für Menschen unsichtbar macht, kann SCP-079 dich noch immer wahrnehmen.
+SCP-268 wird dich nicht vor SCP-096 retten, solltest du ihn angeschaut haben.
 SCP-018 ist nicht vorhersehbar. Nimm Deckung, wenn es umherspringt.
-SCP-018 kann genutzt werden, um Verfolger zu bremsen.
-SCp-914 kann alle Gegenstände verändern, auch Waffen und Munition.
+SCP-018 kann genutzt werden, um Verfolger auszubremsen.
+SCP-330 kann sechs einzigartige Bonbons ausgeben, die alle ihre eigene Wirkung haben. Nimm nicht mehr als zwei!
+SCP-1853 ermöglicht ein schnelleres Nachladen von Waffen und verringert den gefühlten Rückstoß.
+SCP-2176 kann verwendet werden, um einen Raum zu verschließen oder einen Raum oder eine Tür zu öffnen, die von SCP-079 verschlossen wurde.
+SCP-2176 kann SCP-079 lahmlegen, wenn er in einem Raum eingesetzt wird, der gerade beobachtet wird.
```
Comment (Dat-Joseph) [inline Translations/de/LoadingHints.txt:44]: "er" to "es"

File: Translations/de/LoadingHints.txt [hunk 4]
```diff
@@ -1,63 +1,84 @@
...
-SCP-939 kann für eine undefinierte Zeit sprinten und ist komplett lautlos beim Laufen.
-SCP-939 ist lautlos beim Laufen, das macht es zum perfekten Jäger.
-SCP Schritte sind anders als solche von Menschen, hör ganz genau hin!
-SCP-049 kann Menschen sofort töten, sobald sie in seiner Reichweite sind.
+SCP-939 kann die Stimmen von Spielern imitieren, die es zuvor getötet hat.
+SCP-939 kann eine Wolke aus amnestischem Gas freisetzen. Menschen, die unter dieser Wirkung stehen, können SCP-939 nicht wahrnehmen, bis es angreift.
+Die Verwendung von Adrenalin hebt die Wirkung der amnestischen Wolke von SCP-939 auf.
+Blendgranaten sind besonders wirksam gegen SCP-939.
+Je geringer die Ausdauer von SCP-939 ist, desto schlechter kann es andere Spieler sehen.
+Je länger SCP-939 im Fokus ist, desto effektiver wird doe Ultrasensitiv-Fähigkeit.
```
Comment (Dat-Joseph) [inline Translations/de/LoadingHints.txt:75]: "doe" to "die"

File: Translations/de/LoadingHints.txt [hunk 5]
```diff
@@ -1,63 +1,84 @@
...
+SCP Schritte sind anders als die von Menschen, hör ganz genau hin!
```
Comment (Dat-Joseph) [inline Translations/de/LoadingHints.txt:76]: believe "höre" might be better

**Review by Dat-Joseph — APPROVED** [review]

**Pull request merged**